wagtail/admin/urls/workflows.py

```diff
@@ -5,13 +5,13 @@
 
 app_name = 'wagtailadmin_workflows'
 urlpatterns = [
-    path('', workflows.Index.as_view(), name='index'),
-    path('add/', workflows.Create.as_view(), name='add'),
-    path('enable/<int:pk>/', workflows.enable_workflow, name='enable'),
-    path('disable/<int:pk>/', workflows.Disable.as_view(), name='disable'),
-    path('edit/<int:pk>/', workflows.Edit.as_view(), name='edit'),
-    path('remove/<int:page_pk>/', workflows.remove_workflow, name='remove'),
-    path('remove/<int:page_pk>/<int:workflow_pk>/', workflows.remove_workflow, name='remove'),
+    path('workflows/', workflows.Index.as_view(), name='index'),
+    path('workflows/add/', workflows.Create.as_view(), name='add'),
+    path('workflows/enable/<int:pk>/', workflows.enable_workflow, name='enable'),
+    path('workflows/disable/<int:pk>/', workflows.Disable.as_view(), name='disable'),
+    path('workflows/edit/<int:pk>/', workflows.Edit.as_view(), name='edit'),
+    path('workflows/remove/<int:page_pk>/', workflows.remove_workflow, name='remove'),
+    path('workflows/remove/<int:page_pk>/<int:workflow_pk>/', workflows.remove_workflow, name='remove'),
```

Comment:
For consistency, `wagtail/admin/urls/__init__.py` should retain the `workflows/` prefix, and these `workflow/` (since all but the index one apply to one workflow)